src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobMetadataStore.cs

```diff
@@ -122,13 +140,50 @@ await using (Stream stream = _recyclableMemoryStreamManager.GetStream(GetInstanc
         }, cancellationToken);
     }
 
+    // TODO: This should removed once we migrate everything and the global flag is turned on
```

Comment:
When all data is migrated, won't we remove `GetInstanceBlockBlobs`?

Reply:
Yes updated the comment